views/logon.ejs

```diff
@@ -0,0 +1,19 @@
+<%- include("partials/head.ejs") %>
+<%- include("partials/header.ejs")%>
+<form method="POST">
+  <div>
+    <label name="email">Enter your email:</label>
+    <input name="email" />
+  </div>
+  <div>
+    <label name="password">Enter your password:</label>
+    <input type="password" name="password" />
```

Comment:
Add this field <input type="hidden" name="_csrf" value="<%= _csrf %>"> after this line. It's required for csrf validator.